extensions/doctrine/Numeric/Rand.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Dit is niet echt de plaats om zoiets neer te zetten. Een beter idee is om dit in de Application module toe te voegen, of een aparte module aan te maken voor extensies aan andere Modules / libraries.